Lib/concurrent/futures/process.py

```diff
@@ -382,96 +364,139 @@ def shutdown_worker():
         elif wakeup_reader in ready:
             is_broken = False
             result_item = None
```

Comment:
This line doesn't look necessary anymore.

Reply:
Agreed, it seems a bit redundant now if we're also setting it to None on line 356. In any situation where the `elif wakeup_reader in ready` branch is reached, result_item will already be None.